.SRCINFO

```diff
@@ -11,21 +11,18 @@ pkgbase = neomutt
 	makedepends = libxslt
 	depends = openssl
 	depends = gdbm
-	depends = mime-types
+	depends = mailcap
 	depends = libsasl
 	depends = gnupg
 	depends = gpgme
 	depends = libidn
 	depends = krb5
 	depends = notmuch-runtime
 	optdepends = urlview: for url menu
-	provides = mutt
-	conflicts = mutt
-	source = https://github.com/neomutt/neomutt/archive/neomutt-20170912.tar.gz
-	source = https://github.com/neomutt/neomutt/releases/download/neomutt-20170912/neomutt-20170912.tar.gz.sig
-	validpgpkeys = 86C2397270DD7A561263CA4E5FAF0A6EE7371805
-	sha256sums = 9f0d128fe8dbbbddb67e2b629f8b5e52a9d892f82c735845c1708c14e3046657
+	provides = neomutt
+	conflicts = neomutt
+	source = git+https://github.com/neomutt/neomutt.git#branch=master
```

Comment:
No need to specify a branch if `master` is to be used.

Reply:
Duly noted. Will be rectified.